File: others/compare-reductions/otherscripts/utilities/addNewLine.py
#!/usr/bin/python

import os, sys

args = sys.argv
inputDir = args [1]

filenameList = os.listdir (inputDir)

outputDir = "new-%s" % inputDir
os.system ("mkdir %s" %outputDir)
for filename in filenameList: 
    inFilename = "%s/%s" % (inputDir, filename)
    inFile      = open  (inFilename)

    outFilename = "%s/%s" % (outputDir, filename)
    outFile     = open (outFilename, "w")
    for line in inFile:
        outFile.write (line.strip())
        outFile.write ("\n")

        

File: others/compare-reductions/otherscripts/utilities/addNewLine.R

args = commandArgs (T)
inputDir = args [1]

filenameList = list.files (inputDir)

newDir = sprintf ("new-%s", inputDir)
system (sprintf ("mkdir %s", newDir))
for (f in filenameList) {
	file    = read.csv (sprintf ("%s/%s", inputDir, f))
	newFilename = sprintf ("%s/%s", newDir,f)

	write.csv (newFilename, file)
}

